src/sentry/testutils/fixtures.py

```diff
@@ -460,6 +461,23 @@ def create_notification_settings_provider(self, *args, **kwargs):
     def create_user_option(self, *args, **kwargs):
         return Factories.create_user_option(*args, **kwargs)
 
+    def create_monitor(self, **kwargs):
```

Comment:
Can probably be reused in a lot of our tests. We have some monitor base test case that has this too

Reply:
Yeah, ideally we'd just use this one method